dialog_test/flowtest_v1.py

```diff
@@ -27,7 +28,12 @@ class FlowTestV1:
                         'Intent9', 'Confidence9', 'Intent10', 'Confidence10' ]
     
     def __init__(self, **kwargs):
-        self.ctk = ConversationV1(**kwargs)
+        authenticator = IAMAuthenticator(kwargs.get("password"))
+        self.ctk = AssistantV1(
+            version=kwargs.get("version"),
+            authenticator=authenticator
+        )
+        self.ctk.set_service_url("https://gateway.watsonplatform.net/assistant/api")
```

Comment:
The url should be passed as one of the arguments to this function, was it not before? (I recall testing this with US-East URL)

Reply:
Since the ibm_watson library change the auth method to use IAMAuthenticator, it now requires the url.  So a reference wasn't in this script before.  I'll re-examine the args.

Reply:
Passed in the url argument from flowiest.py.